---
title: "Research"
date: 2019-03-08T08:48:38Z
draft: false
---

<img src="heatmaps.jpg" alt="Chromosal expression heatmaps" class="research" />

<p>Gene mutations or environmental factors such as infection or drugs can have genome-wide effects on gene expression levels across many different pathways. However, it is not always clear which of these perturbations can be buffered by the organism's compensatory mechanisms and which lead to detrimental outcomes as reflected in morphological changes or reduced health.</p>

<p>Our research focusses on investigating <i>in vivo</i> gene regulatory responses to challenges such as genetic mutations, chromatin disruption and infection. Specifically, we use our zebrafish mutant archive of ~43,000 alleles covering 60% of protein-coding genes, CRISPR/Cas9 mutagenesis and genome-wide sequencing approaches to understand the relationship between chromatin structure, transcriptional changes and phenotypic outcomes. For example, partial loss of function can have substantial effects on gene expression levels without gross morphological defects. This provides an opportunity to dissect gene regulatory networks and distinguish network-specific, but tolerated, gene expression changes from those that affect organismal development and health. We apply this approach in several areas:</p>

<ul>
    <li>the role of heterochromatin maintenance in development and cancer</li>
    <li>gene regulatory networks underlying genetic compensation during neural crest differentiation</li>
    <li>common and distinct transcriptional responses to loss of skeletal muscle function</li>
</ul>

<p>Furthermore, in collaboration with the laboratory of Lalita Ramakrishnan at the University of Cambridge, we are conducting a genetic screen for loci affecting susceptibility to <i>M. marinum</i> infection using our zebrafish mutant archive. Recently, we have also been awarded an NIH grant, together with Caroline Brennan at Queen Mary University London, to identify genes that influence addiction-related phenotypes in zebrafish.</p>

<h2>Open Science</h2>

<p>As a lab, we are committed to the principles of open science. We share our data and resources as widely as possible, often in advance of publication. For example:<p>

<ul>
    <li>our archive of zebrafish mutant alleles is distributed via the <a href="https://zebrafish.org/">Zebrafish International Resource Center</a> (ZIRC) and/or the <a href="https://www.ezrc.kit.edu/">European Zebrafish Resource Center</a> (EZRC)</li>
    <li>our sequencing data are always deposited in ArrayExpress and/or the European Nucleotide Archive</li>
    <li>our software is always distributed under a free software licence</li>
</ul>

<div class="funders">
    <ul>
        <li><img src="wellcome.png" alt="Wellcome" /></li>
        <li><img src="nih.png" alt="NIH" /></li>
    </ul>
</div>
