---
title: "Resources"
date: 2019-03-08T08:48:38Z
draft: false
---

<p>Here are links to some of the resources we've developed that others might find useful.</p>

<div class="resource">

<h2>ZMP - Zebrafish Mutation Project</h2>

<img src="zmp.png" alt="Zebrafish Mutation Project mutagenesis schematic" />

<div><a href="https://zmp.buschlab.org/">https://zmp.buschlab.org/</a></div>

<p>The Zebrafish Mutation Project at the Wellcome Sanger Institute generated a mutant archive of over 40,000 alleles covering 60% of zebrafish protein-coding genes. The alleles can be ordered from the <a href="https://zebrafish.org/">Zebrafish International Resource Center</a> (ZIRC) and/or <a href="https://www.ezrc.kit.edu/">European Zebrafish Resource Center</a> (EZRC). More information is available in <a href="https://www.nature.com/articles/nature11992">&quot;A systematic genome-wide analysis of zebrafish protein-coding gene function&quot;</a>.</p>

</div>

<div class="resource">

<h2>Zebrafish developmental baseline</h2>

<img src="baseline.png" alt="3D PCA of zebrafish developmental baseline" />

<div><a href="https://www.ebi.ac.uk/gxa/experiments/E-ERAD-475">https://www.ebi.ac.uk/gxa/experiments/E-ERAD-475</a></div>

<p>We have produced an mRNA expression time course of zebrafish development across 18 time points from 1 cell to 5 days post-fertilisation. To make this dataset generally useful, the data can be browsed and downloaded at Expression Atlas and Ensembl. More information is available in <a href="https://elifesciences.org/articles/30860">&quot;A high-resolution mRNA expression time course of embryonic development in zebrafish&quot;</a>.</p>

</div>

<div class="resource">

<h2>Bioinformatics and functional genomics in zebrafish</h2>

<img src="funcgen.png" alt="Bioinformatics and functional genomics in zebrafish" />

<div><a href="https://www.ebi.ac.uk/training/events/bioinformatics-and-functional-genomics-zebrafish-2022/">https://www.ebi.ac.uk/training/events/bioinformatics-and-functional-genomics-zebrafish-2022/</a></div>

<p>Since 2019 our group has been running a bioinformatics and functional genomics course in collaboration with EMBL-EBI and a range of data providers. The aim is to teach researchers how to carry out functional analysis and data visualisation of zebrafish RNA-seq data. All the material for the course is available on the EMBL-EBI website and all of our materials and data for the 2022 iteration are available from <a href="https://funcgen2022.buschlab.org/">https://funcgen2022.buschlab.org/</a>.</p>

</div>

<div class="resource">

<h2>Baseline CompaRe</h2>

<img src="compare.png" alt="Baseline CompaRe count plot" />

<div><a href="https://www.sanger.ac.uk/science/tools/dmdd/dmdd/">https://www.sanger.ac.uk/science/tools/dmdd/dmdd/</a></div>

<p>Baseline CompaRe is an R Shiny app designed to help with the analysis of E8.5-10.5 mouse transcriptomic data, particularly in cases where the embryos of interest are developmentally delayed. We have produced RNA-seq data for wild-type embryos for stages from 4 to 36 somites to form a baseline for comparison to experimental samples. This allows us to prioritise genes that are more likely to be differentially expressed due to the condition of interest, rather than due to developmental delay.</p>

</div>
