---
title: "Software"
date: 2019-03-08T08:48:38Z
draft: false
---

<p>Here are links to some of the open source software we've written that others might find useful.</p>

<h2>RefGenEdit - Reference Genome Editor</h2>

<div><a href="https://github.com/iansealy/RefGenEdit">https://github.com/iansealy/RefGenEdit</a></div>

<p>RefGenEdit is a tool for making edits to a reference genome based on known variation. Reference bias can occur if a sequencing read is more likely to map successfully to the reference allele or to receive a higher quality score after mapping. An edited reference genome, incorporating known variation, can be used to reduce reference bias.</p>

<h2>Baseline CompaRe</h2>

<div><a href="https://github.com/richysix/baseline_compare">https://github.com/richysix/baseline_compare</a></div>

<p>Baseline CompaRe is an R Shiny app designed to help with the analysis of E8.5-10.5 mouse trancriptomics data, particularly in cases where the embryos of interest are developmentally delayed. We have produced RNA-seq data for wild-type embryos for stages from 4 to 36 somites to form a baseline for comparison to experimental samples. This allows us to prioritise genes that are more likely to be differentially expressed due to the condition of interest, rather than due to developmental delay.</p>

<h2>DeTCT - Differential Expression Transcript Counting Technique</h2>

<div><a href="https://github.com/iansealy/DETCT">https://github.com/iansealy/DETCT</a></div>

<p>DeTCT is a pipeline for analysing the 3' end sequencing technique described in <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1788-6">&quot;High-throughput and quantitative genome-wide messenger RNA sequencing for molecular phenotyping&quot;</a> and used in publications like <a href="https://elifesciences.org/articles/30860">&quot;A high-resolution mRNA expression time course of embryonic development in zebrafish&quot;</a>.</p>
